components/src/web-components/input/gs-lineage-filter.tsx

```diff
@@ -45,6 +45,20 @@ export class LineageFilterComponent extends PreactLitAdapter {
     @property()
     lapisField = '';
 
+    /**
+     * The filter to apply to the autocomplete options.
+     * It must be a valid LAPIS filter object.
+     */
+    @property({ type: Object })
+    lapisFilter:
+        | (Record<string, string | string[] | number | null | boolean | undefined> & {
+              nucleotideMutations?: string[];
+              aminoAcidMutations?: string[];
+              nucleotideInsertions?: string[];
+              aminoAcidInsertions?: string[];
+          })
+        | undefined = undefined;
```

Comment:
Nitpick: It would simplify the types a little if we didn't allow `undefined` and had `{}` as the default.
```suggestion
    lapisFilter:
        | (Record<string, string | string[] | number | null | boolean | undefined> & {
              nucleotideMutations?: string[];
              aminoAcidMutations?: string[];
              nucleotideInsertions?: string[];
              aminoAcidInsertions?: string[];
          }) = {};
```

Reply:
I changed it, so the user knows that the default already filters for everything.